Cargo.toml

```diff
@@ -66,6 +66,7 @@ bn254 = { git = "https://github.com/sedaprotocol/bn254" }
 eigen-metrics = { version = "0.0.1-alpha", path = "crates/metrics/" }
 serde = { version = "1.0.197", features = ["derive"] }
 eigen-chainio-utils = { path = "crates/chainio/utils/" }
+eigen-chainio-txmanager = { path = "crates/chainio/txmanager/" }
```

Comment:
alphabetical order, please, here and elsewhere

Reply:
done, thanks!